src/clientfactory.ts

```diff
@@ -14,13 +14,19 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import { Client as DiscordClient, Intents, TextChannel } from "@mx-puppet/better-discord.js";
+import { Client as DiscordClient, GatewayIntentBits, Options, TextChannel } from "discord.js";
 import { DiscordBridgeConfigAuth } from "./config";
 import { DiscordStore } from "./store";
 import { Log } from "./log";
 import { MetricPeg } from "./metrics";
 
 const log = new Log("ClientFactory");
+// `GuildPresences`, `GuildMembers` and `MessageContent` are privileged
+const all_intents: GatewayIntentBits[] = Object.keys(GatewayIntentBits).filter(i => !isNaN(i[0] as any)).map(i => Number(i))
+const intents_non_privileged = all_intents.reduce((a, b) => a + b) -
+    GatewayIntentBits.GuildPresences -
+    GatewayIntentBits.GuildMembers -
+    GatewayIntentBits.MessageContent
```

Comment:
Throughout making this PR, I've felt like it doesn't make sense to expect the bot to lack intents, given what it does and how important those intents are, for example, "MessageContent" is needed to read messages!
discord.js doesn't offer the shorthands `Intents.NON_PRIVILEGED` and `Intents.ALL`, so I had to write such lines that make it explicit what's privileged and not